### PR TITLE
Changes to run on Raspberry Pi

### PR DESCRIPTION
**Pull request by lmandres**
Incorporated modifications by @cdonati to allow install on Raspbian/Wheezy.  Made modifications to allow AppScale to run on Raspberry Pi 2.

### CODE REVIEW

**Comment by menivaitsi** [comment]
Builds:
https://ocd.appscale.com:8080/job/Daily%20Build/1124/
https://ocd.appscale.com:8080/job/Daily%20Build/1125/

File: raspi-bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (cdonati) [inline raspi-bootstrap.sh]: Is this script needed if we merge the other changes?

File: debian/appscale_install_functions.sh [hunk 2]
```diff
@@ -271,7 +280,19 @@ installgems()
     gem install rake ${GEMOPT}
     sleep 1
     # ZK 1.0 breaks our existing code - upgrade later.
-    gem install zookeeper
+    #gem install zookeeper
```
Comment (cdonati) [inline debian/appscale_install_functions.sh]: You can remove this commented-out line.

File: raspi-bootstrap.sh (hunk 1 above)
Comment (lmandres) [inline raspi-bootstrap.sh]: No.  This script is not needed with the other additions.

Comment (cdonati) [inline raspi-bootstrap.sh]: Okay, cool. Thanks for the pull request. I think we can merge it once you remove this file.

**Pull request merged**